### ISSUE TITLE
add `--no-folding`-like option similar to stow

### ISSUE THREAD

**Issue by soredake** [arecarn/dploy#8]
More info on this option can be found here: https://linux.die.net/man/8/stow https://news.ycombinator.com/item?id=7927930

**Comment by harkabeeparolus** [comment]
I agree that this would be a useful feature. Especially for installing personal dotfiles, where you probably want to create a complete directory tree.

**Comment by arecarn** [comment]
yep this makes sense, and see the rationale for not always folding, so that you can add non-stow managed files to a directory in the future.
